lib/viem/getUris.ts

```
@@ -0,0 +1,73 @@
+          contracts: calls as any,
+        });
+        return nfts.map((nft: LatestFeed, i) => {
+          const uri =
+            typeof returnValues?.[i]?.result === "string"
+              ? returnValues[i].result
+              : "";
+          return {
+            ...nft,
+            uri: isNormalizeableIPFSUrl(uri || "")
+              ? uri.replaceAll("{id}", nft.token_id)
+              : Boolean(new URL(uri))
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)